umbra-js/src/utils/utils.ts

```diff
@@ -360,10 +360,16 @@ export async function getEthSweepGasInfo(
   const gasLimitOf21k = [1, 4, 5, 10, 137, 1337]; // networks where ETH sends cost 21000 gas
   const ignoreGasPriceOverride = [10, 42161]; // to maximize ETH sweeps, ignore uer-specified gasPrice overrides
 
-  const [toAddressCode, network, fromBalance, providerGasPrice] = await Promise.all([
+  const [toAddressCode, network, fromBalance, lastBlockData, providerGasPrice] = await Promise.all([
     provider.getCode(to),
     provider.getNetwork(),
     provider.getBalance(from),
+    provider.getBlock('latest'),
+    // getGasPrice is officially deprecated in favor of getFeeData but we use it anyway because:
```

Comment:
nit, since this isn't actually deprecated
```suggestion
    // We use `getGasPrice` instead of `getFeeData` because:
```